src/common/services/git.js

```diff
@@ -69,26 +69,34 @@ function addPostCommitFiles(destination) {
   writePostCommit(destination)
 }
 
-function addPostCommitFilesToSubModules(destination) {
-  if (fs.existsSync(destination)) {
-    for (let submoduleDir of fs.readdirSync(destination)) {
-      const submodulePath = path.join(destination, submoduleDir)
-      const dir = fs.statSync(submodulePath)
+async function getSubmodulesForRepo(repoPath) {
+  const submodulesStatus = await execute('git submodule status', { cwd: repoPath })
+  const statuses = (submodulesStatus && submodulesStatus.trim().split('\n')) || []
 
-      if (dir.isDirectory())
-        addPostCommitFiles(path.join(submodulePath, 'hooks'))
-    }
+  return statuses.map(s => s.trim().split(' ')[1])
+}
+
+async function addPostCommitFilesToSubModules(repoPath) {
+  const submodulesPath = path.join(repoPath, '.git', 'modules')
+
+  if (fs.existsSync(submodulesPath)) {
+    const submodules = await getSubmodulesForRepo(repoPath)
+
+    submodules.map(modulePath => {
```

Comment:
Nitpicky thing, but if we aren't using the value of `.map` we could probably just use `forEach`.

Reply:
Sounds good to me.